### DISCUSSION TITLE
Feature: Update version from program without download from site

### DISCUSSION THREAD

**Discussion by Urvdmih** [doublecmd/doublecmd#1146] [Suggestions]
I would like to see the ability to update the DC directly from the menu, without clicking on the link and manually downloading and installing. To begin with, implement notifications about the release of new versions, and then you can add auto-updates. It would probably be worth adding to the release. I'm talking about the Windows version, because it's easier on Linux.

**Comment by vvch** [comment]
BTW, you may use chocolatey (apt-like package manager for Windows)
https://community.chocolatey.org/packages/doublecmd
https://docs.chocolatey.org/en-us/

when chocolatey installed, to install DC or update DC to latest version just type in command line
```cup doublecmd```

or use Chocolatey GUI

**Comment by leao-nardo** [comment]
yeah . it could at least have a link to the releases page in the about page in the program.
this would be already good enough , actually great .

this link https://github.com/doublecmd/doublecmd/releases

https://github.com/doublecmd/doublecmd/discussions/2114#discussioncomment-11940518 
this comment is excatly as i make too
